src/language/mod.rs

```diff
@@ -3,8 +3,8 @@ mod language;
 pub use language::*;
 
 //pub mod interpreter;
-pub mod interpreter_new;
-pub use interpreter_new as interpreter;
+pub mod interpreter;
+pub use interpreter;
```

Comment:
We can just delete this line; it doesn't do anything anymore. `pub use .. as ..` allowed us to rename the module, but we don't need to do that anymore!